runtime-k3d/src/app_deployment/config/helm/templates/vehicleapp.yaml

```diff
@@ -40,6 +40,9 @@ spec:
           image: "{{ .Values.imageVehicleApp.repository }}:{{ .Values.imageVehicleApp.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.imageVehicleApp.pullPolicy }}
           env:
-          # required for C++ SDK
+          # required for C++ SDK based apps
+          - name: SDV_MQTT_ADDRESS
```

Comment:
Why can we not just keep name as it was before?

Reply:
You mean `MQTT_BROKER_URI`?
Because for being in sync with Python-SDK, where we (with native middleware) define addresses of all services this way: "SDV_[service-name]_ADDRESS", i.e. in case of MQTT broker it is `SDV_MQTT_ADDRESS` (whereat I would prefer `SDV_MQTTBROKER_ADDRESS`)

Reply:
@BjoernAtBosch can you create an issue to rename it to `SDV_MQTTBROKER_ADDRESS`? I would also prefer that name. For now we can live with an alignment between C++ and Python SDKs. Long term we should get this name cleaned up.

Reply:
Issue created: https://github.com/eclipse-velocitas/vehicle-app-python-sdk/issues/90